ledfx/devices/__init__.py

```
@@ -621,8 +621,11 @@ def create_from_config(self, config):
                     config=device["config"],
                     ledfx=self._ledfx,
                 )
-            except vol.MultipleInvalid as e:
-                _LOGGER.exception(e)
+            except Exception as e:
+                # be very prolific on ignoring devices if they are bad
+                _LOGGER.exception(
+                    f"Failed to load device {device.get('id', 'unknown')}: {e}"
```

Ruff (0.14.3)

Redundant exception object included in `logging.exception` call

(TRY401)